test/fork/airstream/CampaignLL.t.sol

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.19 <0.9.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { Lockup, LockupLinear } from "@sablier/v2-core/src/types/DataTypes.sol";
+
+import { ISablierV2AirstreamCampaignLL } from "src/interfaces/ISablierV2AirstreamCampaignLL.sol";
+
+import { MerkleBuilder } from "../../utils/MerkleBuilder.sol";
+import { Fork_Test } from "../Fork.t.sol";
+
+abstract contract AirstreamCampaignLL_Fork_Test is Fork_Test {
+    constructor(IERC20 asset_) Fork_Test(asset_) { }
+
+    function setUp() public virtual override {
+        Fork_Test.setUp();
+    }
+
+    /// @dev Encapsulates the data needed to compute a Merkle tree leaf.
+    struct LeafData {
+        uint256 index;
+        address recipient;
+        uint128 amount;
+    }
+
+    struct Params {
+        address admin;
+        LeafData[] leafData;
+        uint256 leafPos;
+    }
+
+    struct Vars {
+        uint256 actualAirstreamId;
+        LockupLinear.Stream actualStream;
+        uint128[] amounts;
+        ISablierV2AirstreamCampaignLL campaignLL;
+        uint256 campaignTotalAmount;
+        uint128 clawbackAmount;
+        uint256 dataCount;
+        uint256 expectedAirstreamId;
+        address expectedCampaignLL;
+        LockupLinear.Stream expectedStream;
+        uint256[] indexes;
+        bytes32[] leaves;
+        bytes32 merkleRoot;
+        address[] recipients;
+    }
+
+    function testForkFuzz_ClaimHasClaimedClawback(Params memory params) external {
+        vm.assume(params.admin != users.admin.addr);
+        vm.assume(params.leafData.length > 250);
```

Comment:
I wanted to have as many leaves as possible, but it wouldn't have worked if it was >256.

Reply:
I don't understand what the benefit is.

1. Given enough fuzz runs, the fuzzer will naturally generate values in the upper ranges
2. Why would we want to restrict the fuzzing to a narrow range - the whole point of fuzzing is to test the whole gamut of values.

I will change this to `params.leafData.length > 0`.

Reply:
The benefit is that it will be closer to reality, airstream campaigns will usually have a high number of claimants.

Reply:
It will be closer to reality even with the `> 0` check. Again, that's what the fuzzer does - it covers the whole range.

This is guaranteed to be the case in CI where we have a very high number of fuzz runs, because, as you say, the upper limit is 256.

Reply:
Ok, will push a commit now, it has to be `> 1` because a Merkle tree can't have a length of 1.